contracts/Optimism_SpokePool.sol

```diff
@@ -23,11 +23,11 @@ contract Optimism_SpokePool is CrossDomainEnabled, SpokePoolInterface, SpokePool
         address _crossDomainAdmin,
         address _hubPool,
         address _wethAddress,
-        uint64 _depositQuoteTimeBuffer,
-        address timerAddress
+        address timerAddress,
+        uint32 _depositQuoteTimeBuffer
```

Comment:
should we default this to what we use in prod (10 mins) then remove it from the constructor? this is what I did in the hub pool: as much as posible remove the instantiation of constructor variables if we know what they will be in production. thoughts?

Reply:
yea lets do it